### PR TITLE
Glossary/concepts work continuation for 2.10.21 sprint

### PR DESCRIPTION
**Pull request by GiselleLeeOSI**

### CODE REVIEW

**Comment by GiselleLeeOSI** [comment]
@digitaljoshua, all conflicts have been resolved. We can work off of this branch in the upcoming sprint. If needed, we can meet Thursday instead of Friday as scheduled. I'll fill you in on what had been done.

File: Content_Portal/Documentation/Assets/asset-centric-data-api.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ uid: AssetCentricDataAPI
 
 # Asset Centric Data API
 The asset centric data API provides a quick way to retrieve data stored in an asset's referenced streams.
+The asset centric data API provides a quick way to take the dynamic data stored in streams and store it as references within a given asset measurement mappings.
```
Comment (GiselleLeeOSI) [inline Content_Portal/Documentation/Assets/asset-centric-data-api.md:7]: @lasato-OSI, please take a look at the line and choose what you think is appropriate.

File: Content_Portal/Documentation/Identity/Identity_Invitation.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ Invitations are issued after the creation of a User object in OCS,
             date is extended the User can accept an Invitation.
             Invitations that are past of their expiration date by more than two weeks will be deleted.
             The only way to provision a User after this, is to send a new Invitation.
+Invitations are issued after the creation of a User object in OCS, to begin the provisioning process for a user with one of the identity providers in a tenant. There can only be one invitation for a user at a time. An invitation can expire, at which time the admin can either delete it, or extend it. When creating an invitation an account admin has the option to resend the invitation email to the ContactEmail configured for the user the invitation is attached to. The email can be re-sent by updating the invitation. When an invitation expires the user cannot accept it. If the expiration date is extended the User can accept an invitation. Invitations that are past of their expiration date by more than two weeks will be deleted. The only way to provision a user after this, is to send a new invitation.
```
Comment (GiselleLeeOSI) [inline Content_Portal/Documentation/Identity/Identity_Invitation.md:19]: Worth checking @digitaljoshua

File: Content_Portal/Documentation/Identity/Identity_PersistedGrant.md [hunk 3]
```diff
@@ -4,11 +4,7 @@ uid: identityPersistedGrant
 
 # PersistedGrant
 
-Persisted Grants such as refresh tokens or authorization codes are used
-            by OSIsoft Cloud Services to maintain some state about User authentication with regards to
-            a Client. For instance, refresh tokens, which are issued to Hybrid clients,
-            make it possible to determine when the Client can be issued a new access token
-            on behalf of the User without the users themselves being present.
+Persisted grants, such as refresh tokens or authorization codes, are used by OSIsoft Cloud Services to maintain some state about User authentication with regards to a client. For instance, refresh tokens, which are issued to hybrid clients, make it possible to determine when the client can be issued a new access token on behalf of the User without the users themselves being present.
```
Comment (GiselleLeeOSI) [inline Content_Portal/Documentation/Identity/Identity_PersistedGrant.md:7]: worth checking

File: Content_Portal/Documentation/Identity/Identity_User.md [hunk 4]
```diff
@@ -18,6 +18,8 @@ Users consume resources in a Tenant. They are invited by the Admin of the
             upon creation and can be modified by an Admin. We allow the change of some User fields
             and the complete deletion of a User.
             For some recommendations on least privilege for users and clients, refer to the [Least Privilege](xref:LeastPrivilege) topic.
+Users consume resources in a tenant. They are invited by the admin of the tenant and should already have an account in one of the configured identity providers for this tenant. A user is fully provisioned in OSIsoft Cloud Services only after they have accepted the invitation and successfully logged with an identity provider. OSIsoft Cloud Services does not maintain user credentials, but it delegates authentication to the identity provider the user logged in with at first. Once
```
Comment (GiselleLeeOSI) [inline Content_Portal/Documentation/Identity/Identity_User.md:21]: please check @digitaljoshua

File: Content_Portal/Concepts/token-management.md [hunk 5]
```diff
@@ -5,13 +5,12 @@ uid: TokenManagement
 # Token Management
 The token that results from a successful authentication process and contains some predefined fields. This is sent as part of the Authorization header with all requests to OSIsoft resources and services. Clients can be configured to generate Access Tokens with varying expiration between 60 seconds and one hour. The default Access Token lifespan is one hour. There is no limit on the number of Access Tokens a Client can be issued at any given time. 
 
-## ClientCredential Clients
+## Client Credential Clients
```
Comment (GiselleLeeOSI) [inline Content_Portal/Concepts/token-management.md:8]: @digitaljoshua, you might want to check if spacing is okay here.

File: Content_Portal/Concepts/credential-management.md [hunk 6]
```diff
@@ -2,20 +2,19 @@
...
 
-# Credential Management
+# Credential management
 
 ## Users
 User authentication and credential management is delegated to the external Identity Providers.
 
-## Clients
-Client secrets which are used to authenticate a Client are generated by OCS for Client Credentials. Each Client can have a maximum of 10 secrets at any given time. OSIsoft suggests following a secret rotation pattern. Secrets have a description and expiration date.
+## Client secrets
```
Comment (GiselleLeeOSI) [inline Content_Portal/Concepts/credential-management.md:10]: @digitaljoshua, please take a look select the final wording.

File: Content_Portal/Concepts/token-management.md [hunk 7]
```diff
@@ -5,13 +5,12 @@ uid: TokenManagement
 # Token Management
 The token that results from a successful authentication process and contains some predefined fields. This is sent as part of the Authorization header with all requests to OSIsoft resources and services. Clients can be configured to generate Access Tokens with varying expiration between 60 seconds and one hour. The default Access Token lifespan is one hour. There is no limit on the number of Access Tokens a Client can be issued at any given time. 
 
-## ClientCredential Clients
```
Comment (GiselleLeeOSI) [inline Content_Portal/Concepts/token-management.md:8]: @digitaljoshua, please confirm if changing the title is appropriate.

File: Content_Portal/How_To_Guides/Assets/asset-type-overview.md [hunk 8]
```diff
@@ -4,19 +4,22 @@ uid: AssetTypes
 
 # Asset types
 
-Asset types are an asset template that you use to create multiple assets with similar properties. Note the following characteristics of an asset created from an asset type:  
+Asset Types define a template for building assets. Asset types contain:
```
Comment (GiselleLeeOSI) [inline Content_Portal/How_To_Guides/Assets/asset-type-overview.md:7]: @lasato-OSI, please take a look at these edits Eric made in the topic and accept or change as you see appropriate.

Comment (lasato-OSI) [reply Content_Portal/How_To_Guides/Assets/asset-type-overview.md:7]: Please use this wording:
Asset types are a template that is used to create multiple assets with similar properties. Assets created from asset types have the following characteristics:

Thanks for checking with me.

File: Content_Portal/Concepts/token-management.md (hunk 7 above)
Comment (digitaljoshua) [reply Content_Portal/Concepts/token-management.md:8]: Correct

File: Content_Portal/Concepts/token-management.md (hunk 5 above)
Comment (digitaljoshua) [reply Content_Portal/Concepts/token-management.md:8]: Correct

File: Content_Portal/Documentation/Identity/Identity_AuthorizationCodeClient.md [hunk 9]
```diff
@@ -84,10 +84,9 @@ All responses will have an error message in the body. The exceptions are 200 res
 
 If and when contacting OSIsoft support about this error, please provide the OperationId.
 
-## `Create an Authorization Code Client`
+## `Create an authorization code client`
```
Comment (GiselleLeeOSI) [inline Content_Portal/Documentation/Identity/Identity_AuthorizationCodeClient.md:87]: @digitaljoshua, this is an instance of us keeping the original line with uppercase content in an API name.

File: Content_Portal/Documentation/Identity/Identity_AuthorizationCodeClient.md [hunk 10]
```diff
@@ -197,7 +196,7 @@ Created.
 
 #### 400
 
-Missing or invalid inputs, or Client limit exceeded.
+Missing or invalid inputs or client limit exceeded.
```
Comment (GiselleLeeOSI) [inline Content_Portal/Documentation/Identity/Identity_AuthorizationCodeClient.md:199]: This you probably want to keep Eric's edit @digitaljoshua .

File: Content_Portal/Documentation/Identity/Identity_PersistedGrant.md (hunk 3 above)
Comment (digitaljoshua) [reply Content_Portal/Documentation/Identity/Identity_PersistedGrant.md]: just cleaning up the indentation from waht i can see...OK.

File: Content_Portal/Documentation/Identity/Identity_Invitation.md (hunk 2 above)
Comment (digitaljoshua) [reply Content_Portal/Documentation/Identity/Identity_Invitation.md:19]: Just cleaning up indentation in the code. OK

File: Content_Portal/Documentation/Identity/Identity_User.md (hunk 4 above)
Comment (digitaljoshua) [reply Content_Portal/Documentation/Identity/Identity_User.md:21]: Looks like cleaning up indentation in the code. OK.

File: Content_Portal/Documentation/Identity/Identity_AuthorizationCodeClient.md (hunk 10 above)
Comment (digitaljoshua) [reply Content_Portal/Documentation/Identity/Identity_AuthorizationCodeClient.md:199]: agreed

File: Content_Portal/Concepts/credential-management.md (hunk 6 above)
Comment (digitaljoshua) [reply Content_Portal/Concepts/credential-management.md:10]: There is no guiding principle to this as of yet. We can finalize this with Joyce.

File: Content_Portal/Documentation/Identity/Identity_AuthorizationCodeClient.md (hunk 9 above)
Comment (digitaljoshua) [reply Content_Portal/Documentation/Identity/Identity_AuthorizationCodeClient.md]: i had to return all of the API names back to initial caps

**Comment by digitaljoshua** [comment]
I am done with the initial work to merge Eric's original work into PR 333.

**Pull request merged**